stdlib/src/posixsubprocess.rs

```diff
@@ -92,6 +92,7 @@ fn exec(args: &ForkExecArgs, procargs: ProcArgs) -> ! {
             let _ = write!(cur, "OSError:{}:", e as i32);
             let pos = cur.position();
             let _ = unistd::write(args.errpipe_write, &buf[..pos as usize]);
+            // TODO: Need to change to ExitCode or Termination
```

Comment:
I don't think this is expected to be ExitCode because it doesn't raise SystemExit